addons/pad/py_etherpad/__init__.py

```diff
@@ -179,7 +181,7 @@ def setText(self, padID, text):
     def setHtmlFallbackText(self, padID, html):
         try:
             # Prevents malformed HTML errors
-            html_wellformed = '<html><body>' + html + '</body></html>'
+            html_wellformed = Markup("<html><body>") + html + Markup("</body></html>")
```

Comment:
Alternatively:
```suggestion
            html_wellformed = Markup("<html><body>%s</body></html>") % html
```
Bigger change but, I think, for the better.

Reply:
@xmo-odoo thank you